src/todolist.js

```diff
@@ -0,0 +1,110 @@
+export default class TodoList {
+a
+
+constructor() {
+  const savedTasks = JSON.parse(localStorage.getItem('tasks'));
+  if (savedTasks) {
+    this.tasks = savedTasks;
+  } else {
+    this.tasks = [];
+  }
+}
+
+displayTasks() {
+  this.sortAndSave();
+  const check = 'fa-check';
+  const uncheck = 'fa-square-o';
+  const lineThrough = 'linethrough';
+  const taskSection = document.querySelector('.task-list');
+  taskSection.innerHTML = '';
+  this.tasks.forEach((task) => {
+    const Done = task.completed ? check : uncheck;
+    const Line = task.completed ? lineThrough : '';
+    const taskTemp = `
```

Comment:
- [ ] It is good practice to use more descriptive names. In this case, the Done, Line, and taskTemp variable names can be changed as follows:
   - `Done` -- > `taskStatus`
   - `Line` --> `taskDescriptionTextDecoration`
   - `taskTemp` --> `newTaskListItem`